cfgov/v1/signals.py

```diff
@@ -72,8 +79,12 @@ def invalidate_filterable_list_caches(sender, **kwargs):
     # Get the cache backend and purge filterable list page cache tags if this
     # page belongs to any
     if len(cache_tags_to_purge) > 0:
-        cache_backend = configure_akamai_backend()
```

Comment:
With this change we can also delete the `configure_akamai_backend` method left behind on line 102 of this file.

Reply:
Removed.